test/functional/eval/system_spec.lua

```diff
@@ -273,7 +273,7 @@ describe('system()', function()
 
   describe('input passed as Number', function()
     it('stringifies the input', function()
```

Comment:
This test isn't valid anymore.  The test description (`stringifies the input`) is invalidated by the fact that a number is now considered to be a buffer number.

If the `describe('input passed as Number')` block is to be kept, then the test needs to be updated to properly test the effects of a number.

Reply:
@ckelsel Will you be able to address this soon?

Reply:
@jamessan Can I delete this test `input passed as Number`

Reply:
Since `test_system.vim` covers `system('wc -l', bufnr('%'))`, yes, it's ok to delete `describe('input passed as Number...`.

Reply:
@ckelsel Let's just change the test description:

    it('treated as a buffer number', function()